### PR TITLE
Infinispan fixes and obsolete APIs replacement.

### PR DESCRIPTION
**Pull request by zebrik**
@rhatlapa , @honza-kasik WDYT? As soon as there will be agreement on API I will continue with refactoring of remaining Infinispan tests.

### CODE REVIEW

**Comment by zebrik** [comment]
ConfigChecker build separated to builder to enhance readability as requested.

File: common/src/main/java/org/jboss/hal/testsuite/util/ConfigChecker.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+        public Builder(OnlineManagementClient client, Address resourceAddress) {
+            this.client = client;
+            this.resourceAddress = resourceAddress;
+        }
+
+        /**
+         * setter for {@link ConfigFragment}
+         */
+        public Builder config(ConfigFragment config) {
```
Comment (rhatlapa) [inline common/src/main/java/org/jboss/hal/testsuite/util/ConfigChecker.java]: Maybe, I would change the name to configFragment to make it more clear that the UI element is expected.

**Comment by rhatlapa** [comment]
Just minor comment, otherwise LGTM

**Comment by zebrik** [comment]
OK, method rename force pushed.

**Pull request merged**